ElDorito/Source/ElDorito.cpp

```diff
@@ -306,6 +308,11 @@ void ElDorito::OnMainMenuShown()
 		Server::DedicatedServer::Init();
 	else
 		Web::Ui::ScreenLayer::Show("title", "{}");
+
+	if (firstShow) {
+		// Initialize Discord
+		Discord::DiscordRPC::Instance();
+	}
```

Comment:
This is unneeded, it will be initialized when calling update here https://github.com/ElDewrito/ElDorito/blob/master/ElDorito/Source/ElDorito.cpp#L263